src/main/features/response/views/full-admission/payment-plan.njk

```diff
@@ -51,10 +54,19 @@
         }
         ]
         ) }}
-
       </div>
 
-      {{ textArea(label = t('Explain why you can’t pay the full amount now'), name = 'text', form = form) }}
+      {{ linkButton('Calculate length of repayment plan', 'action[calculatePaymentPlan]', 'button button-secondary calculateLengthOfRepayment') }}
+
+      <h2 class="heading-small">Length of repayment plan</h2>
+      <p id="lengthOfRepayment" class="lengthOfRepayment"> {{ paymentLength | default("-") }} </p>
```

Comment:
Please use `'-'` for default value. We tend to use single quotes to represent JS / Nunjucks values. We use `"` only for HTML.

Reply:
done